src/super_gradients/training/models/prediction_pose_estimation_results.py

```diff
@@ -0,0 +1,321 @@
+import os
+from dataclasses import dataclass
+from typing import List
+
+import numpy as np
+
+from super_gradients.training.models.prediction_results import ImagePrediction, ImagesPredictions, VideoPredictions
+from super_gradients.training.models.predictions import PoseEstimationPrediction
+from super_gradients.training.utils.media.image import show_image, save_image
+from super_gradients.training.utils.media.video import show_video_from_frames, save_video
+from super_gradients.training.utils.visualization.poses import draw_skeleton
+
+
+@dataclass
+class ImagePoseEstimationPrediction(ImagePrediction):
+    """Object wrapping an image and a detection model's prediction.
+
+    :attr image:        Input image
+    :attr predictions:  Predictions of the model
+    :attr class_names:  List of the class names to predict
+    """
+
+    image: np.ndarray
+    prediction: PoseEstimationPrediction
+
+    def draw(
+        self,
+        joint_colors=None,
+        joint_thickness: int = 2,
+        keypoint_colors=None,
+        keypoint_radius: int = 5,
+        box_thickness: int = 2,
+        show_confidence: bool = True,
+    ) -> np.ndarray:
+        """Draw the predicted bboxes on the image.
+
+        :param joint_colors:    Optional list of tuples representing the colors for each joint.
+                                If None, default colors are used.
+                                If not None the length must be equal to the number of joint links in the skeleton.
+        :param joint_thickness: Thickness of the joint links  (in pixels).
+        :param keypoint_colors: Optional list of tuples representing the colors for each keypoint.
+                                If None, default colors are used.
+                                If not None the length must be equal to the number of joints in the skeleton.
+        :param keypoint_radius: Radius of the keypoints (in pixels).
+        :param show_confidence: Whether to show confidence scores on the image.
+        :param box_thickness:   Thickness of bounding boxes.
```

Comment:
can we also drow without boxes?

Reply:
Yes, we can draw without bounding boxes, once can achieve that by passing `show_confidence=False`.
The goal of drawing bounding box is to help the user understand to which pose this confidence text belongs to.

Reply:
However I agree that drawing a box makes a picture cluttered. I changed a default value to False (e.g not showing boxes by default)

![With boxes](https://github.com/Deci-AI/super-gradients/assets/532320/804ad7f6-c1fa-4907-b73c-4b94ccf6ae3d)

![No boxes](https://github.com/Deci-AI/super-gradients/assets/532320/9b0e17c3-f44a-4755-9f22-60f640a95af8)